community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/commands/indexQuery.scala

```diff
@@ -52,7 +52,7 @@ object indexQuery extends GraphElementPropertyFunctions {
     case RangeQueryExpression(rangeWrapper) =>
       val range = rangeWrapper match {
         // StringSeekRange => PrefixRange("Petra")
```

Comment:
Can we take this opportunity to get rid of the thins that has been commented out?